lib/planet.rb

```diff
@@ -0,0 +1,29 @@
+# Create class for Planet
+
+class Planet
+   # To avoid repition use attr_reader
+  attr_reader(:name, :color, :mass_kg, :distance_from_the_sun_km, :fun_fact)
```

Comment:
With Ruby we normally don't put parentheses around the fields sent to `attr_reader`